docs/sphinx/about/whats-new.txt

```diff
@@ -9,14 +9,14 @@ New features/API:
 * added support for JPEG-XR compressed CZI data (funded by a
   `partnership between Glencoe Software and ZEISS <http://glencoesoftware.com/2016-08-30-glencoe-software-zeiss-partner-open-source-file-reader-whole-slide.html>`_), adding 'ome:jxrlib' as a new dependency
   of Bio-Formats
-* tile-based image writing
+* improved tile-based image writing
     - added new methods to the ``loci.formats.IFormatWriter`` interface
       allowing to set and retrieve the tile along the X and Y dimensions
     - added default implementations to the ``loci.formats.FormatWriter``
       abstract class defaulting to the entire image width/height
     - added functionality to ``loci.formats.TiffWriter`` allowing to write 
```

Comment:
Nods, it does not read well. Does https://github.com/openmicroscopy/bioformats/pull/2687/commits/0c03ded077883b64d44ca25b86fa31437dc8f6a4 make more sense?